src/LinAlg/hiopMatrixDenseRowMajor.cpp

```diff
@@ -371,15 +371,13 @@ void hiopMatrixDenseRowMajor::print(FILE* f,
   }
 }
 
-#include <unistd.h>
-
 /*  y = beta * y + alpha * this * x  
  *
  * Sizes: y is m_local_, x is n_local_, the matrix is m_local_ x n_global_, and the
  * local chunk is m_local_ x n_local_ 
 */
-void hiopMatrixDenseRowMajor::timesVec(double beta, hiopVector& y_,
-			       double alpha, const hiopVector& x_) const
+void hiopMatrixDenseRowMajor::
+timesVec(double beta, hiopVector& y_, double alpha, const hiopVector& x_) const
```

Comment:
Each argument on a separate line, i.e., using style guide.
Same comment applied to the other functions in this file.

Reply:
this is actually valid, arguments can stay on the same line as long as the line is not too long... styleguide needs maybe to be updated to make this more explicit